Tests/Marketplace/Tests/prepare_private_id_set_for_merge_test.py

```diff
@@ -54,10 +74,16 @@ def test_remove_old_pack_from_private_id_set():
     - ensure that in case there is no pack name no error returns, and the ID set remains as it is
     """
     private_id_set = remove_old_pack_from_private_id_set(PRIVATE_ID_SET, 'Workday')
-    assert WORKDAY_PACK not in private_id_set['integrations']
+    assert WORKDAY_INTEGRATION not in private_id_set['integrations']
+    assert 'Workday' not in list(private_id_set.get('Packs').keys())
+    assert "WORKDAY" not in list(private_id_set.get('Packs').keys())
 
     private_id_set = remove_old_pack_from_private_id_set(PRIVATE_ID_SET, 'Accessdata')
-    assert ACCESSDATA_PACK not in private_id_set['integrations']
+    assert ACCESSDATA_INTEGRATION not in private_id_set['integrations']
+    assert "Accessdata" not in list(private_id_set.get('Packs').keys())
+    assert "Access data" not in list(private_id_set.get('Packs').keys())
 
     private_id_set = remove_old_pack_from_private_id_set(PRIVATE_ID_SET, '')
-    assert ACTIVEMQ_PACK in private_id_set['integrations']
+    assert ACTIVEMQ_INTEGRATION in private_id_set['integrations']
+    assert "ActiveMQ" in list(private_id_set.get('Packs').keys())
+    assert "Active MQ" in list(private_id_set.get('Packs').keys())
```

Comment:
Let's try to enhance the readability of this test.
Currently, it's a bit hard to understand what is being tested

Reply:
done